lms/djangoapps/experiments/utils.py

```diff
@@ -69,6 +74,39 @@ def get_experiment_user_metadata_context(course, user):
         if enrollment.is_active:
             enrollment_mode = enrollment.mode
             enrollment_time = enrollment.created
+
+            # TODO: clean up as part of REVEM-106 (START)
+            # get program data for this course
+            request = get_current_request()
+            if request:
+                enrollment_list = [enrollment]
+                meter = ProgramProgressMeter(request.site, user, enrollments=enrollment_list)
+                if meter.engaged_programs and meter.engaged_programs[0]:
+                    org_name = None
+                    courses_not_started = 0
+                    courses_in_progress = 0
+                    courses_completed = 0
+                    program_data = meter.engaged_programs[0]
+                    program_data = ProgramDataExtender(program_data, user, mobile_only=False).extend()
+                    program_orgs = program_data.get('credit_backing_organizations')
+                    if program_orgs and program_orgs[0]:
+                        org = program_orgs[0]
+                        org_name = org.get('name')
+                    if meter.progress() and meter.progress()[0]:
+                        progress = meter.progress()[0]
+                        courses_not_started = progress.get('not_started')
+                        courses_in_progress = progress.get('in_progress')
+                        courses_completed = progress.get('completed')
+                    program_key = {
```

Comment:
i don't have any context on this ticket, but i'm not sure if this answers these bullet points in the ticket
1. Whether user is enrolled in all courses in that program
2. # of this course in that program